src/main/java/io/jenkins/pluginhealth/scoring/scores/DeprecatedPluginScoring.java

```diff
@@ -37,10 +39,15 @@ public class DeprecatedPluginScoring extends Scoring {
     private static final float COEFFICIENT = .8f;
     private static final String KEY = "deprecation";
 
+
+
     @Override
     protected ScoreResult doApply(Plugin plugin) {
         final ProbeResult deprecatedPluginProbeResult = plugin.getDetails().get(DeprecatedPluginProbe.KEY);
-        if (deprecatedPluginProbeResult == null || deprecatedPluginProbeResult.status().equals(ResultStatus.FAILURE)) {
+
```

Comment:
```suggestion
```

Reply:
please revert your change from this class as finally we won't touch this `DeprecatedPluginScoring` class.